superduperdb/backends/mongodb/query.py

```diff
@@ -534,7 +534,21 @@ def execute(self, db):
             self.table_or_collection.identifier
         )
         if self.one:
-            return collection.delete_one(*self.args, **self.kwargs)
+            ids = []
+            if '_id' in self.kwargs:
+                ids = [str(self.kwargs['_id'])]
+            for arg in self.args:
+                if isinstance(arg, dict) and '_id' in arg:
+                    ids = [str(arg['_id'])]
+            result = collection.delete_one(*self.args, **self.kwargs)
+            if result.deleted_count == 1:
+                if not ids:
+                    deleted_document = collection.find_one(*self.args, **self.kwargs)
+                    return [deleted_document['_id']]
+                return ids
+            else:
+                return []
```

Comment:
After converting to str, is it compatible with ObjectID? If the id in the database is ObjectID, the data cannot be queried using str.

Reply:
so these deleted_ids are being consumed by us in the program so I think it won't have any effect